node/node_test.go

```diff
@@ -611,7 +611,8 @@ func TestNodeInvalidNodeInfoCustomReactors(t *testing.T) {
 }
 
 func TestSaveAndLoadBigGensisFile(t *testing.T) {
-	stateDB := dbm.NewMemDB()
+	stateDB, err := dbm.NewDB("state", "goleveldb", os.TempDir())
```

Comment:
I'm not sure but:
* Is only using `goleveldb` enough?
* Can't we use `dbm.NewMemDB()` for this test?

Reply:
Of course, we can also test using `memdb`. However, since the ostracon node uses `goleveldb` as the default setting, only goleveldb tests have been added.

Reply:
> the ostracon node uses `goleveldb` as the default setting

You're right. But, basically the `Ostracon` test uses `memdb` in the test. If you want to use the `goleveldb` for the test, please write down it in the PR description. I agree with using `goleveldb`. Because a large size file test should use with `goleveldb` for disk IO related test. (I think it's not related the default setting.)

JFI:
* There are some test cases using `goleveldb`:
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/p2p/trust/store_test.go#L24
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/state/txindex/kv/kv_test.go#L337

* There is a `dbm.NewRocksDB`:
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/abci/example/kvstore/persistent_kvstore.go#L41
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/cmd/ostracon/commands/light.go#L122
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/light/example_test.go#L49
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/light/example_test.go#L117
  * https://github.com/line/ostracon/blob/cc6bd1307edfc98f181cdd8f407a651dfd363cf8/state/txindex/kv/kv_bench_test.go#L24

Reply:
Yes, it would be better to specify that I will be testing only using goleveldb. I have corrected the code.